packages/core/src/App/Containers/CloseMXAccountModal/close-mx-account-modal.jsx

```diff
@@ -100,9 +132,10 @@ const CloseMXAccountModal = ({
 
     return (
         <div className='close-mx-account'>
-            <Dialog is_visible={is_visible} is_loading={is_loading}>
+            <Dialog is_visible={is_visible} is_loading={is_loading} is_uk={is_uk} is_eu={is_eu}>
                 <CloseMXAccountContent
-                    is_iom={is_iom}
+                    is_uk={is_uk}
+                    is_eu={is_eu}
```

Comment:
(question) why are we replacing `iom` with `uk` check here? :thinking:

Reply:
Since the notification only pops up if has_malta_account or has_iom_account is true, I think it means that only iom, uk and eu malta account users can see the modal.

So I thought, writing code like this should work:

if (is_uk) {
 ...iom uk account text...
} else if (standpoint.malta) {
...eu malta account text...
} else {
...iom account text...
}

Hence, I didn't need is_iom check anymore.